lib/endpoints/class-wp-rest-comments-controller.php

```diff
@@ -73,12 +76,36 @@ public function register_routes() {
 				'callback' => array( $this, 'update_item' ),
 				'permission_callback' => array( $this, 'update_item_permissions_check' ),
 				'args'     => array(
-					'post'         => array(),
-					'status'       => array(),
-					'content'      => array(),
-					'author'       => array(),
-					'author_email' => array(),
-					'author_url'   => array(),
+					'post'         => array(
```

Comment:
@joehoyle We also cannot set a `default` argument in the schema.  Feels like setting the endpoint arguments from the schema is too rigid at this point to fit all our routes.